README.md

```diff
@@ -139,10 +127,10 @@ VMware ESXi:
 - 6.0 u2
 - 6.5
 
-Docker: 1.9 and higher
+Docker: 1.9 and higher (Recommended 1.13/17.03 and above to use managed plugin)
```

Comment:
Change this to Docker 1.12 and higher

Also move this line at the end of GOS section